stable/artifactory-ha/README.md

```diff
@@ -440,6 +440,7 @@ artifactory:
   userPluginSecrets:
     - archive-old-artifacts
     - webhook
+    - '{{ template "my-chart.fullname" . }}'
```

Comment:
Can you add a short explanation of this use case. This mix in the example is confusing.